app/analytics/query.ts

```diff
@@ -277,23 +287,23 @@ export class AnalyticsEngineAPI {
                         counts.views += Number(row.count);
                     });
                     resolve(counts);
-                })(),
+                })()
         );
 
         return returnPromise;
     }
 
-    async getVisitorCountByColumn(
+    async getVisitorCountByColumn<T extends keyof typeof ColumnMappings>(
```

Comment:
This is the only real change (other than the runtime checks) from the previous PR. Introducing a generic here means that the return type can be inferred based on which column is passed.